src/main/java/com/vfasad/controller/ClientController.java

```diff
@@ -14,11 +21,65 @@
     @Autowired
     private ClientService clientService;
 
+    @Autowired
+    private UserService userService;
+
     @GetMapping("/clients")
     @Secured(ROLE_ADMIN)
     public ModelAndView users() {
         ModelAndView model = new ModelAndView("client/clients");
         model.addObject("clients", clientService.getClients());
         return model;
     }
+
+    @RequestMapping(value = "/clients/add", method = RequestMethod.GET)
+    @Secured(ROLE_ADMIN)
+    public ModelAndView addUserForm() {
```

Comment:
Method name doesn't matter for Spring MVC, but matters for human code reader, so let's keep it actual :)